.github/workflows/publish-code.yml

```diff
@@ -58,10 +58,10 @@ jobs:
         run: dotnet restore
       - name: Build solution [Release]
         working-directory: src
-        run: dotnet build --no-restore -c Release -p:Version=$ref
+        run: dotnet build --no-restore -c Release -p:Version=$version
       - name: Pack solution [Release]
         working-directory: src
-        run: dotnet pack --no-restore -c Release -p:Version=$ref -o out
+        run: dotnet pack --no-restore --no-build -c Release -p:Version=$version -o out
```

Comment:
Why was there no `--no-build` ?

Reply:
Probably because "if it ain't broke, don't fix it"... lol